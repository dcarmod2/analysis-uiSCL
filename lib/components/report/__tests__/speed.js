//
import enzyme from 'enzyme'
import React from 'react'

import Speed from '../speed'

describe('Report > Speed', () => {
  it('renders correctly', () => {
    const props = {
      kmh: 2000
    }

    // mount component
    const tree = enzyme.shallow(<Speed {...props} />)
    expect(tree).toMatchSnapshot()
  })
})
