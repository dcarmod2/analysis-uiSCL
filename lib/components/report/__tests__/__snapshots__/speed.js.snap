// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Report > Speed renders correctly 1`] = `
<Fragment>
  2000 km/h
   (
  1243 mph
  )
</Fragment>
`;
